fca_giorgio.dbc

```diff
@@ -0,0 +1,236 @@
+ SG_ COUNTER : 43|4@0+ (1,0) [0|15] "" XXX
+ SG_ CHECKSUM : 55|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 263 ABS_4: 8 ABS
+ SG_ BRAKE_PRESSURE : 7|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 265 NEW_MSG_109: 8 XXX
+ SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
+ SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 270 ABS_7: 7 XXX
+ SG_ LONG_ACCEL_RAW : 7|12@0+ (0.01,-20.48) [0|4095] "" XXX
+ SG_ LATERAL_ACCEL_RAW : 11|12@0+ (0.01,-20.48) [0|4095] "" XXX
+ SG_ YAW_RATE_RAW : 31|12@0+ (-0.0014,2.86) [0|4095] "" XXX
+ SG_ COUNTER : 43|4@0+ (1,0) [0|15] "" XXX
+ SG_ CHECKSUM : 55|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 278 ABS_5: 8 ABS
+ SG_ WHEEL_IMPULSE_FL : 7|8@0+ (1,0) [0|255] "" XXX
+ SG_ WHEEL_IMPULSE_FR : 15|8@0+ (1,0) [0|255] "" XXX
+ SG_ WHEEL_IMPULSE_RL : 23|8@0+ (1,0) [0|255] "" XXX
+ SG_ WHEEL_IMPULSE_RR : 31|8@0+ (1,0) [0|255] "" XXX
+ SG_ ACTIVE_FL : 32|1@0+ (1,0) [0|1] "" XXX
+ SG_ ACTIVE_FR : 33|1@0+ (1,0) [0|1] "" XXX
+ SG_ ACTIVE_RL : 34|1@0+ (1,0) [0|1] "" XXX
+ SG_ ACTIVE_RR : 35|1@0+ (1,0) [0|1] "" XXX
+ SG_ FORWARD_1 : 36|1@0+ (1,0) [0|1] "" XXX
+ SG_ REVERSE_1 : 37|1@0+ (1,0) [0|1] "" XXX
+ SG_ FORWARD_2 : 38|1@0+ (1,0) [0|1] "" XXX
+ SG_ REVERSE_2 : 39|1@0+ (1,0) [0|1] "" XXX
+ SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
+ SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 280 NEW_MSG_118: 6 XXX
+ SG_ COUNTER : 35|4@0+ (1,0) [0|15] "" XXX
+ SG_ CHECKSUM : 47|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 282 NEW_MSG_11A: 8 XXX
+ SG_ NEW_SIGNAL_1 : 7|11@0+ (1,-1000) [0|2047] "" XXX
+ SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
+ SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 284 NEW_MSG_11C: 8 XXX
+ SG_ NEW_SIGNAL_1 : 7|12@0+ (1,0) [0|4095] "" XXX
+ SG_ NEW_SIGNAL_2 : 11|12@0+ (1,0) [0|4095] "" XXX
+ SG_ NEW_SIGNAL_3 : 28|13@0+ (1,0) [0|8191] "" XXX
+ SG_ NEW_SIGNAL_4 : 31|2@0+ (1,0) [0|3] "" XXX
+ SG_ VEHICLE_SPEED : 47|12@0+ (0.017,0) [0|4095] "m/s" XXX
+ SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
+ SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 288 NEW_MSG_120: 6 XXX
+ SG_ COUNTER : 19|4@0+ (1,0) [0|15] "" XXX
+ SG_ CHECKSUM : 31|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 290 EPS_3: 4 EPS
+ SG_ EPS_TORQUE : 7|12@0+ (1,-2048) [0|4095] "" XXX
+ SG_ COUNTER : 19|4@0+ (1,0) [0|15] "" XXX
+ SG_ CHECKSUM : 31|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 501 NEW_MSG_1F5: 5 XXX
+ SG_ NEW_SIGNAL_2 : 22|1@0+ (1,0) [0|1] "" XXX
+ SG_ NEW_SIGNAL_3 : 27|4@0+ (1,0) [0|15] "" XXX
+ SG_ NEW_SIGNAL_1 : 39|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 502 LKA_COMMAND: 8 CAMERA
+ SG_ LKA_TORQUE : 7|11@0+ (1,-1024) [0|2047] "" XXX
+ SG_ HAPTIC_WARN_1 : 8|1@0+ (1,0) [0|1] "" XXX
+ SG_ LKA_ACTIVE : 11|1@0+ (1,0) [0|1] "" XXX
+ SG_ NEW_SIGNAL_1 : 12|1@0+ (1,0) [0|1] "" XXX
+ SG_ HAPTIC_WARN_2 : 19|1@0+ (1,0) [0|1] "" XXX
+ SG_ COUNTER : 51|4@0+ (1,0) [0|15] "" XXX
+ SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 508 NEW_MSG_1FC: 8 XXX
+ SG_ NEW_SIGNAL_1 : 6|1@0+ (1,0) [0|1] "" XXX
+
+BO_ 601 NEW_MSG_259: 8 XXX
+ SG_ NEW_SIGNAL_1 : 47|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 762 NEW_MSG_2FA: 3 XXX
```

Comment:
According to my Giulia, 0x2fa should be....  

 SG_ BTN_CRUISE : 1|1@0+ (1,0) [0|255] "" XXX
   SG_ BTN_RES : 7|1@0+ (1,0) [0|1] "" XXX
   SG_ BTN_SET_DUAL_DOWN : 5|3@0+ (1,0) [0|1] "" XXX
   SG_ BTN_SET_DUAL_UP : 5|3@0+ (1,0) [0|1] "" XXX
   SG_ BTN_SET_SINGLE_DOWN : 5|3@0+ (1,0) [0|255] "" XXX
   SG_ BTN_SET_SINGLE_UP : 5|3@0+ (1,0) [0|255] "" XXX
   SG_ BUTTONS_STATUS : 7|8@0+ (1,0) [0|255] "" XXX




VAL_ 762 BTN_CRUISE 1 "PRESSED" 0 "NOT_PRESSED";
VAL_ 762 BTN_RES 1 "PRESSED" 0 "NOT_PRESSED";
VAL_ 762 BTN_SET_DUAL_DOWN 4 "PRESSED" 3 "NOT_PRESSED" 2 "NOT_PRESSED" 1 "NOT_PRESSED" 0 "NOT_PRESSED";
VAL_ 762 BTN_SET_DUAL_UP 2 "NOT_PRESSED" 0 "PRESSED";
VAL_ 762 BTN_SET_SINGLE_DOWN 3 "PRESSED" 0 "NOT_PRESSED" 2 "NOT_PRESSED" 1 "NOT_PRESSED";
VAL_ 762 BTN_SET_SINGLE_UP 1 "PRESSED" 0 "NOT_PRESSED" 2 "NOT_PRESSED" 3 "NOT_PRESSED";
VAL_ 762 BUTTONS_STATUS 144 "RES" 18 "CRUISE" 8 "SET_UP_ONEPRESS" 0 "SET_UP_STEP2" 24 "SET_DOWN_ONEPRESS" 32 "SET_DOWN_STEP2" 16 "IDLE";


;-)